docs/alerts/alert-variables.md

```diff
@@ -36,9 +36,13 @@ Variables must be enclosed by double curly brackets (`{{ }}`). Unresolved variab
 | `{{TriggerTimeEnd}}` | The end time of the time range that triggered the monitor in Unix format. For example, `1626190592042`. | &#9989;| &#9989;|
 | `{{SourceURL}}` | The URL to the configuration or status page of the monitor in Sumo Logic. | &#9989;| &#10060; |
 | `{{AlertResponseUrl}}` | When your Monitor is triggered it will generate a URL and provide it as the value of this variable where you can use it to open Alert Response. | &#9989;| &#10060; |
-| `{{AlertName}}` | Name of the Alert that will be displayed on the Alert page.  | &#9989;| &#9989;|
+| `{{AlertName}}` | Name of the Alert that will be displayed on the Alert page. | &#9989;| &#9989;|
 | `{{Playbook}}` | Allows you to access the [playbook content](/docs/alerts/monitors#metrics-trigger-types) that was configured as part of the initial monitor setup. | &#9989;| &#9989;|
 
+:::caution
+When writing queries, do not define fields with built-in field names; their values will be overridden.
```

Comment:
It would be nice if we add a link to the list of build in fields so users can quickly refer to.

Reply:
Those fields are listed above this note

Reply:
The fields we are worried about the using are listed here https://help.sumologic.com/docs/search/get-started-with-search/search-basics/built-in-metadata/ . A further complication is these fields have a short hand name. In this example "_sourceName" can also be refered to with "Name". We probably need to improve the documentation on the built-in-metadata page to include these shorthand names as well

Reply:
@dstephens5 , thank you for the tip! Where can I find the rest of the shorthand names for that built-in metadata table? I put in some placeholders for now... https://github.com/SumoLogic/sumologic-documentation/pull/1286/files